x/docs/md/cli/windows.md

```diff
@@ -16,7 +16,7 @@ If you already have the Chocolatey Package Manager installed, feel free to skip
 Chocolately lets you install Windows applications quickly and easily. To install Chocolatey please refer to the [installation instructions on the Chocolatey website](https://chocolatey.org/install).
 
 > **NOTE**
-> Please ensure that when you open up your Windows shell, whether it's the Command Prompt or Powershell, you open it with **Administrator privileges**, otherwise the Chocolatey installation will fail. If you are unsure how to do this, please refer to the [help below](#open-win-cmd).
+> Please ensure that when you open up your Windows shell, whether it's the Command Prompt or Powershell, you open it with **Administrator privileges**, otherwise the Chocolatey installation will fail. If you are unsure how to do this, please refer to the [help below](#open-win-cmd). If you are not an Administrator on your PC then it is recommended that you have Chocolatey and the Exercism Languages installs performed by someone that is an Administrator on the PC.  
```

Comment:
✨ Thanks!

I don't think that Chocolatey is used to install any programming languages, but rather the Exercism CLI. I think only Chocolatey needs to be installed with admin privileges—once Chocolatey is in place, I *think* (but am not certain) that the Exercism CLI can be installed with normal privileges.

@NobbZ do you happen to know the details here?

Reply:
Erlang does suggest to use `choco` and does not provide alternative instructions, since erlang via choco “just works”. No fiddling with the `%PATH%` or anything else, just running it once. Also upgrading is pretty easy and does not result in parallel installation of multiple erlangs, which can happen easily when installing them by hand.

Also, regardless of the fact that another language might suggest choco or not to install their tools, these tools might need to be installed as administrators.

At work I have to use windows for a couple of things and I am administrator there, and I can't remember a single install program that didn't ask for elevation… Oh well, there was one, but that had to be elevated manually, perhaps some legacy thing, its been for a software that was already old when Win95 came out…

And even if there is software that is able to get installed without elevation, in scenarios where a user is not able to elevate, there is also often corporate guidelines or rules that forbid usage of external software in general. Often enough its near to impossible to only put in some USB stick to copy over some presentation prepared at home, because of the fear of viruses…

Reply:
Ok, in that case, let's simply remove _and the Exercism Languages_, and adjust the sentence to read well without it.